### PR TITLE
bfd-1153- added a synthea properties file and excerpt to design data sets

### PR DESCRIPTION
**Pull request by cbrunefearless**
### Change Details
Added the synthea.properities file as well as added the parameters i ran the synthea utility to generate the 10000 records.

### Acceptance Validation

Completeness of design-data-sets

### Feedback Requested
Look at the synthea properities files and make sure the parameters are correct for synthea data sets

### External References

- [BFD-1153](https://jira.cms.gov/browse/BFD-1153)

### Security Implications

- [ ] new software dependencies
N/A

- [ ] altered security controls
N/A

N/A
- [ ] new data stored or transmitted

N/A
- [ ] security checklist is completed for this change

- [ ] requires more information or team discussion to evaluate security implications

### CODE REVIEW

**Review by mackec1 — DISMISSED** [review]
Looks reasonable; did not match props to code, but expect that to be more on the synthea generation side.

File: apps/bfd-model/bfd-model-rif-samples/dev/design-sample-data-sets.md [hunk 1]
```diff
@@ -513,4 +513,20 @@ data-synthetic/Sample-A-Synthetic-Data/ based on data from Step 1
...
+exporter.bfd.bene_id_start = -10000000000000
+exporter.bfd.clm_id_start = -10000000000000
+exporter.bfd.clm_grp_id_start = -100000000
+exporter.bfd.pde_id_start = -10000000000000
+exporter.bfd.mbi_start = 1S00-E00-AA00
+exporter.bfd.hicn_start = T01000000A
+exporter.bfd.partd_contract_start = Z0001
+exporter.bfd.partd_contract_count = 10
+
+Ran the synthea utility with the following command: ./run_synthea -s 0 -cs 0 -r 20210808 -p 10000 --exporter.fhir.export=false --exporter.bfd.export=true --exporter.years_of_history=10 --generate.only_alive_patients=true -a 65-80
```
Comment (karlmdavis) [inline apps/bfd-model/bfd-model-rif-samples/dev/design-sample-data-sets.md:532]:
```suggestion
Ran the synthea utility with the following command:

    ./run_synthea -s 0 -cs 0 -r 20210808 -p 10000 --exporter.fhir.export=false --exporter.bfd.export=true --exporter.years_of_history=10 --generate.only_alive_patients=true -a 65-80
```

File: apps/bfd-model/bfd-model-rif-samples/dev/design-sample-data-sets.md [hunk 2]
```diff
@@ -513,4 +513,20 @@ data-synthetic/Sample-A-Synthetic-Data/ based on data from Step 1
...
+exporter.bfd.export = true
+exporter.bfd.config_file = export/bfd_field_values.tsv
+exporter.bfd.bene_id_start = -10000000000000
+exporter.bfd.clm_id_start = -10000000000000
+exporter.bfd.clm_grp_id_start = -100000000
+exporter.bfd.pde_id_start = -10000000000000
+exporter.bfd.mbi_start = 1S00-E00-AA00
+exporter.bfd.hicn_start = T01000000A
+exporter.bfd.partd_contract_start = Z0001
+exporter.bfd.partd_contract_count = 10
```
Comment (karlmdavis) [inline apps/bfd-model/bfd-model-rif-samples/dev/design-sample-data-sets.md:530]: I'd just link to the file directly, like this:

```suggestion
09/09/2021 - Generated Synthea data set from the following properities file that is located at: `[./synthea/synthea.properities-08092021](./synthea/synthea.properities-08092021)`.
```

File: apps/bfd-model/bfd-model-rif-samples/dev/synthea/synthea.properities-08092021 [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+exporter.use_uuid_filenames = false
+exporter.subfolders_by_id_substring = false
+# number of years of history to keep in exported records, anything older than this may be filtered out
+# set years_of_history = 0 to skip filtering altogether and keep the entire history
+exporter.years_of_history = 10
+# split records allows patients to have one record per provider organization
+exporter.split_records = false
+exporter.split_records.duplicate_data = false
+exporter.ccda.export = false
+exporter.fhir.export = true
```
Comment (karlmdavis) [inline apps/bfd-model/bfd-model-rif-samples/dev/synthea/synthea.properities-08092021:13]: @jawalonoski / @hadleynet : Do we actually need/want this? And would you mind reviewing the other properties in here? You understand them much better than I.

**Review by karlmdavis — CHANGES_REQUESTED** [review]
@cbrunefearless Love this -- glad to see we're trying to save this documentation and config.

Just a couple small suggestions.

**Review by christrotteradhoc — APPROVED** [review]

**Review by jzulim — APPROVED** [review]
Comments addressed. Looks like a reasonable manner to record this. Nice work.

**Pull request merged**